src/renderer/components/Experiment/Train/TrainingModalDataTemplatingTab.tsx

```diff
@@ -306,7 +308,7 @@ function TrainingModalDataTemplatingTab({
         flexDirection: 'column',
       }}
     >
-      {chatTemplateData?.data && (
+      {parsedData?.supports_chat_template && (
```

Comment:
I thought we have a concept of "supports" as an array that is a primary key on plugins. Shouldn't we be using that?